docs/topics/modelling_considerations.md

```diff
@@ -0,0 +1,194 @@
+
+What does it mean that those objects are Context Entities (instead of classes, objects, tables, or documents). The
+difference lays in how it is treated around the platform: modifications of the CEs can be observed by subscriptions, the
+history of those changes stored, rules can be defined to modify the global Context based on CEs changes.
+
+All the given examples have several things in common:
+
+* All of them model nouns: Context Entities are subjects of interactions; they have a history, they may have subscriptions
+to their changes, etc. This behavior does not fit well with processes (and things typically modelled as verbs or actions).
+
+* All those entities are expected to have a long life in the system. Entities are not supposed to model
+transient information (e.g.: support tickets, mails, log files). This doesn't mean short-lived entities can't be modelled
+in the system, but having those kind of entities in your project will heavily impact the performance
+
+## The Platform is not a ticketing system
+
+Another common mistake involves the use of the Platform for short-lived ticketing, as in the following scenarios:
+
+* A system that keeps track of alarms risen by sensors, and whether they have been solved or not. Modelled entities are:
+Sensor and Alarm.
+
+* A booking system for Theater seats, modelling each ticket as a Context Entity. Modelled entities are: Seat and Ticket.
+
+In both cases there is a Context Entity that does not comply with the above rules about Context Entities: Ticket and Alarm.
+Both entities model short lived elements that will be created once and again in the system, cluttering your service with
+old information.
+
+Objects like tickets, even being nouns, model interactions between entities in your system instead of modelling contextual
+elements: a ticket is a temporal association between a user (that may go once and again to the same theater) and the seat
+(where he will be seated in a particular play). A more convenient way to model this kind of information would be to
+directly code it in the seat, as in the following example:
+
+      {
+         "type": "Seat",
+         "isPattern": "false",
+         "id": "Row12SeatB",
+         "attributes": [
+           {
+             "name": "status",
+             "type": "String",
+             "value": "occupied"
+           },
+           {
+             "name": "play",
+             "type": "String",
+             "value": "Richard III"
+           },
+           {
+             "name": "audience",
+             "type": "Person",
+             "value": {
+                "name": "Bob Smith"
+             }
+           },
+         ]
+      }
+
+With this modelling, the Context Broker will be storing the actual picture of the Theater: which seats have been sold and
+which seats haven't. The history of each seat could also be made available (by means of the STH), giving information of
```

Comment:
"...by means of STH and other history persistence backend" (as STH is not the only mechanism to store history).